controllers/details.js

```diff
@@ -4,7 +4,8 @@ var api = require('../api');
 
 module.exports = function renderDetails(req, res) {
   if (req.params && req.params.sysnum) {
-    api.requestID(req.params.sysnum)
+    const username = req.headers['X-Auth-Username'] || req.headers['x-auth-username'];
```

Comment:
I think we should either just use the first version and be case sensitive. Or we should do something for proper case-insensitive matching (probably involving underscore / lodash).